patterns/2-structured/core-team.md

```diff
@@ -0,0 +1,102 @@
+## Title
+
+Core Team
+
+## Patlet
+
+Is your InnerSource project difficult for people to work with, yet no one is interested in improving it?
+Establish a core team to take care of the project's fundamental items so that contributors are enabled to add and use the features that provide value to their scenarios.
+
+## Problem
+
+* It is difficult to contribute to the project.
+This could be due to things like:
+  * Can't run the project locally.
+  * Poor documentation.
+  * Convoluted code.
+  * Inadequate testing.
+* It is difficult to use the project.
+Some possible causes:
+  * Poor documentation (again).
+  * Frequent bugs.
+  * Unintuitive setup.
+
+## Story
+
+There's a central project that everyone depends on.
+What a great candidate for InnerSource!
+Unfortunately, the project has grown organically, with various contributions and additions slapped on haphazardly.
+Now it's an icky, black morass of code that no one understands and everyone is afraid to touch.
+It's clearly due for an overhaul, but even though everyone needs that and wants it, no one takes the time to work on it.
+
+## Context
+
+* Many teams use the project.
+* Significant tech debt.
+* Slow adoption and iteration on the project.
+
```

Comment:
I think part of the context is that there is no  core team responsible for the project OR there may be a project owner, but they are not taking responsibility for organizing and promoting contributions to reduce technical debt, etc.

Reply:
I agree. 

If what @NewMexicoKid is describing is indeed the case, then I would expect that to see repeated in multiple places e.g. Patlet + Problem + Forces

Reply:
Added a bullet point for this feedback, although I'm not sure if it will meet the gist of what you're trying to share?

Reply:
I still have some understanding issues with this patterns. It feels like there are some unspoken assumptions.
The following questions are meant to uncover those:

1. If somebody had asked "who owns/maintains this project", before the Core Team solution was in place, what would the answer have be?
2. Is this Pattern the same as saying "every project should have a maintainer (maintaining team)"?
3. What are the alternatives to get out of this situation, other than a Core Team?

Reply:
1.  I think this is in the **Context** now.  "Nobody".
2.  It is one way to move out of the given **Context**.  If someone has this **Context** and wants to move out of it then they can apply this pattern.  If they don't have this **Context** or don't want to move out of it then they don't have to.
3.  I don't know.  If someone had an alternative then they could write another pattern with the same **Context** and a different **Solution**?

Reply:
Sebastian, for me the context is pretty clear and doesn't need to be specified further. It could match situations where literally no one group or person owns the project (e.g., perhaps someone left the company or changed roles) or where there are nominal owners but not effectively driving the projects anymore (though perhaps that might take a different pattern like a "Friendly Takeover").
I think that a Core Team is probably a good recommendation/pattern for any IS project that fulfills a wide need by many other teams and that isn't well organized/maintained (i.e., matches the context). It is possible there are alternate patterns/solutions to this context, but that isn't a problem for this pattern to solve (though if one later springs up, it could be mentioned as an alternative).

Reply:
Just to give more context about my questions above:

I didn't mean to suggest that this should not be a pattern. It absolutely should be a pattern!
I was just asking these questions because I wanted to confirm why the project doesn't have an owner/maintainer already, given that it is "widely needed".

I have suggested a modification to L38 that would make this more clear to me.

Is possible that in addition to "widely needed" this project is actually already "widely used" as well, i.e. many teams already depend on it? That would also explain the mentioned contributors that only fix their own needs.

I am further suspecting that this project had an owner/maintainer/core team in the past, which got the project to a state where it could become "widely used". However that owner/maintainer/core team somehow vanished?

Some of this extra info could go into the **Story**, or possibly into the **Rationale**.
@NewMexicoKid is making good points above how a project may end up in an "un-owned" and "hard-to-contribute-to" situation, and I suspect many InnerSource practitioners will be able to related to this situation, as it happens everywhere. Still wouldn't hurt to share some examples of why/how this may happen.